test/pom.xml

```diff
@@ -220,6 +220,11 @@
       <artifactId>accumulo-iterator-test-harness</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This should already be on the class path. It's brought in transitively through accumulo-monitor, and we shouldn't have any direct dependencies on it, only transitive runtime dependencies on it. The test module has a direct dependency on the accumulo-monitor.

Ultimately, this should be fixed by 2.19.1, which should re-add the transitive dependency for log4j-slf4j2-impl (https://issues.apache.org/jira/browse/LOG4J2-3601), but it hasn't been done yet, and some of the devs seemed confused about why they needed to fix that. So, I don't know when it will happen.

If this helps fix something during testing in an IDE, I'm fine with adding it. It's not going to hurt.